CONTRIBUTING.md

```diff
@@ -33,6 +33,9 @@ All members of the Istio community must abide by the
 [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 Only by respecting each other can we develop a productive, collaborative community.
 
+Please be sure to read and understand our 1 page
+[shared values](VALUES.md) summary.
```

Comment:
I think the Contributing is getting long (and correctly includes a lot of details) - I think having a 1 pager specifically about shared understanding and values is more effective?